pkgs/applications/science/electronics/nanovna-saver/default.nix

```diff
@@ -1,41 +1,38 @@
 {
   lib,
-  python3,
+  python311,
   fetchFromGitHub,
   fetchpatch,
-  wrapQtAppsHook,
+  qt6,
 }:
-python3.pkgs.buildPythonApplication rec {
+python311.pkgs.buildPythonApplication rec {
   pname = "nanovna-saver";
-  version = "0.5.4";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
     owner = "NanoVNA-Saver";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-CLfgDQt2rOXtWwvEhlXEstPp28nFhuhiAPYL6EjZVu4=";
+    sha256 = "sha256-2vDjAdEL8eNje5bm/1m+Fdi+PCGxpXwpxe2KvlLYB58=";
   };
 
-  # Fix for https://github.com/NanoVNA-Saver/nanovna-saver/issues/579
-  # Try dropping the patch in the next release after v0.5.4
-  patches = [
-    (fetchpatch {
-      name = "remote-changelog-from-setup-py.patch";
-      url = "https://github.com/NanoVNA-Saver/${pname}/commit/d654ea0441939e4e1c599d1333b587a185394fbe.diff";
-      sha256 = "sha256-ifOhiWD0EYyQZRKp2W3G6crmWslca+/21APmhpfP/xE=";
-    })
+   nativeBuildInputs = [
+    qt6.wrapQtAppsHook
+    qt6.qtbase
   ];
 
-  nativeBuildInputs = [ wrapQtAppsHook ];
-
-  propagatedBuildInputs = with python3.pkgs; [
+  propagatedBuildInputs = with python311.pkgs; [
     cython
     scipy
-    pyqt5
+    pyqt6
     pyserial
     numpy
+    setuptools
+    setuptools_scm
```

Comment:
For some reason, this input causes `nixpkgs-review` to fail during evaluation (`x86_64-linux`):
```
$ nixpkgs-review pr 223068
[...]
error: undefined variable 'setuptools_scm'
```
Would you mind investigating this? Perhaps it is due to a recent change on master. Or perhaps I'm doing something wrong here, but I can't think of anything that would cause a simple nixpkgs-review invocation to fail.